src/test/java/com/atlassian/db/replica/api/TestDualConnection.java

```diff
@@ -793,27 +796,27 @@ public void shouldShowThatReadRunAfterWriteIsRunOnMainNotReplica() throws SQLExc
         verify(stateListener).transition(State.REPLICA, State.MAIN);
         verifyNoMoreInteractions(stateListener);
         verify(databaseCall).call(
-                any(),
```

Comment:
@wyrzyk Can you please share (can be DM  message), your's formatting options, previously I've added this code and looks that we have different settings for it.

Reply:
We can check if our IDEs all support `.editorconfig`. It's on by default in IntelliJ. Is it enabled in your IDE @wyrzyk ?
We can reformat the entire project in a single PR. Then each of us can checkout that branch, reformat the entire project and expect an empty diff.

Reply:
> We can check if our IDEs all support .editorconfig.

Do you mean this option? 

![Zrzut ekranu 2021-04-21 o 09 15 48](https://user-images.githubusercontent.com/3447570/115512342-2b401400-a282-11eb-8d94-76b44a04f141.png)

Reply:
> Can you please share (can be DM message), your's formatting options


@afaruga-atlassian Do you mean this?

```
<code_scheme name="Default" version="173">
  <codeStyleSettings language="JAVA">
    <indentOptions>
      <option name="CONTINUATION_INDENT_SIZE" value="4" />
    </indentOptions>
  </codeStyleSettings>
</code_scheme>
```

Reply:
![Screenshot 2021-04-21 at 10 44 08](https://user-images.githubusercontent.com/77149850/115524510-824be600-a28e-11eb-8d1e-f58cf53e48b6.png)


I've checked `.editorconfig`
```
indent_size = 4
```
I didn't have plugin EditorForConfig enabled, maybe that's why it didn't worked. 
Onced enabled it nicely prompt:
![Screenshot 2021-04-21 at 10 46 56](https://user-images.githubusercontent.com/77149850/115524909-e53d7d00-a28e-11eb-8794-3bdb80630e2e.png)

Thanks!

Reply:
CONTINUATION_INDENT_SIZE != indent_size